pyqtgraph/parametertree/parameterTypes/basetypes.py

```diff
@@ -1,11 +1,10 @@
 import builtins
+from functools import  partial
```

Comment:
do we still use this?

Reply:
Yes, its being used here in `GroupParameterItem` and is being used to always return the original font size: The current font size is returned and incremented by 1 for items where depth=0. Since `updateDepth` is now run every time(to paint the backgrounds), when switching from dark/light system themes, the font size would increase 1 point, every time the theme is switched without it(or something similar). 

```
    def __init__(self, param, depth):
        ParameterItem.__init__(self, param, depth)
        # prevent 0 depth item's font from growing on every call of updateDepth
        self.fontPointSize = partial(self.font(0).pointSize)
```

Reply:
Would it have been the same as ```self.fontPointSize = self.font(0).pointSize```?

Reply:
> Would it have been the same as self.fontPointSize = self.font(0).pointSize?

The behavior would have been the same. The difference to me is, when an attribute is defined as in `__init__` generally you wouldn't expect that its required to be called later. Seeing it defined in a lambda, or with partial lets you know that it is a callable.

Reply:
I think this isn't used any more with the most recent commit; once this import is removed, I think we can merge this 🎉